ptvsd/wrapper.py

```diff
@@ -2326,6 +2339,11 @@ def on_setDebuggerProperty(self, request, args):
 
         self.send_response(request)
 
+    def is_vs_client(self):
+        """ Return True if the client is Visual Studio 2017 False otherwise
```

Comment:
You may want just to say 'Visual Studio' since this code should be working in Dev17+